src/assets/scripts/resources-utils.js

```diff
@@ -268,12 +268,12 @@ function renderResources(resources, resourceTopics, resourceTypes) { // eslint-d
 		resourcesHtml += `
 				</div>
 				<div class='card-description'>
-					<p>${resource.description}</p>
+					<p>${escapeSpecialCharactersForHTML(resource.description)}</p>
 				</div>
-				${resource.publishedYear ? `<div class='card-publishedYear'><p>Published in ${resource.publishedYear}</p></div>` : ''}
-				<div class='card-link'><a rel='external' href='${resource.link}'>Visit ${resource.title}${resourceLink.host === hostURL ? '' : '<svg role="presentation"><use xlink:href="#external" /></svg>'}</a></div>
+				${resource.publishedYear ? `<div class='card-publishedYear'><p>Published in ${escapeSpecialCharactersForHTML(resource.publishedYear)}</p></div>` : ''}
+				<div class='card-link'><a rel='external' href='${escapeSpecialCharactersForHTML(resource.link)}'>Visit ${escapeSpecialCharactersForHTML(resource.title)}${resourceLink.host === hostURL ? '' : '<svg role="presentation"><use xlink:href="#external" /></svg>'}</a></div>
```

Comment:
Escaping special chars for html should only be applied to entities that will be displayed as html, such as title, description etc. This function should not be applied to locations where text are not rendered as html, such as:

1. `href` value should be url encoded: whitespace -> %20. I don't think it makes sense to apply this escaping function to it;
2. Other tag attributes are not html too including `alt` attribute;
3. Moreover, when you quote alt text in double quotes, you will need to think about what would happen if the alt text itself contains double quotes. Setting `altText === "a thumbnail"` breaks the code at my end. See the screenshot below.

<img width="447" alt="Screenshot 2023-04-11 at 2 10 08 PM" src="https://user-images.githubusercontent.com/572877/231312249-b947d710-947c-4692-87d0-b2605c8581be.png">

Please think through different cases when applying escaping. Thanks.